processing/src/test/java/io/druid/query/timeseries/TimeseriesQueryRunnerTest.java

```diff
@@ -2562,7 +2562,31 @@ public void testTimeSeriesWithSelectionFilterLookupExtractionFn()
         optimizedRunner.run(query, CONTEXT),
         Lists.<Result<TimeseriesResultValue>>newArrayList()
     );
-    TestHelper.assertExpectedResults(expectedResults, results2);
+    // post aggs are calculated in FinalizeResultsQueryRunner.run()
+    // toolChest.mergeResults doesn't do post aggs, so addRowsIndexConstant is not included in expectedResults2
+    List<Result<TimeseriesResultValue>> expectedResults2 = Arrays.asList(
```

Comment:
Instead of skipping the test for addRowsIndexConstant, please wrap this in FinalizeResultsQueryRunner.